CHANGELOG.md

```diff
@@ -54,6 +54,18 @@ Other Changes
 - Lepton was upgraded to the latest version (PR #349)
 - Made Object::print a const member function (PR #191)
 - Improved the testOptimization/OptimizationExample to reduce the runtime (PR #416)
+- Added shouldBeParallelIfPossible() override methods to all computationally
+expensive muscle force classes
```

Comment:
We discussed this at some length over at simbody:

![image](https://cloud.githubusercontent.com/assets/846001/9191079/cb30d042-3fb1-11e5-8a75-1ae6e46e1d16.png)

![image](https://cloud.githubusercontent.com/assets/846001/9191086/d8b0dfdc-3fb1-11e5-983b-73e2526e3077.png)


I'd give a link but the conversation is in an outdated diff. Here's the PR though:  https://github.com/simbody/simbody/pull/414